### ISSUE TITLE
need to install Microsoft .NET developer pack to build

### ISSUE THREAD

**Issue by waffel** [PowerShell/Phosphor#25]
I needed to install the Microsoft .NET Framework 4.5.1 Developer Pack to successful build the DLL. Else you get an error like 

`
D:\Dev\projects\git\Phosphor\.dotnet\sdk\1.0.0\Microsoft.Common.CurrentVersion.targets(1111,5): error MSB3644: The reference assemblies for framework ".NETFramework,Versi
on=v4.5.1" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for wh
ich you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemb
lies. Therefore your assembly may not be correctly targeted for the framework you intend. [D:\Dev\projects\git\Phosphor\src\Phosphor\Phosphor.csproj]
D:\Dev\projects\git\Phosphor\.dotnet\sdk\1.0.0\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(92,5): error MSB4018: The "GenerateDepsFile" task failed unexpectedl
y. [D:\Dev\projects\git\Phosphor\src\Phosphor\Phosphor.csproj]
`

I would be nice to state this as a requirement in the README to build.

**Comment by daviwil** [comment]
Hmmm...  That shouldn't be needed since the new .NET SDK tools should be self-contained.  I'll look into it and see why this error might be getting raised.  Thanks!

**Comment by daviwil** [comment]
@waffel Was this what you installed?  https://www.microsoft.com/en-us/download/details.aspx?id=40772

**Comment by waffel** [comment]
@daviwil yes .. exactly this was it for my version. I had checked which was installed on my box and installed then this package. After that it creates the DLL.

Side note: I need to use this "older" version because this is forced by my company to use.

**Comment by daviwil** [comment]
Thanks for confirming!
